dvc/render/vega_converter.py

```diff
@@ -76,20 +76,14 @@ def _append_index(datapoints: List[Dict]) -> List[Dict]:
     return datapoints
 
 
-class Converter:
+class VegaConverter:
     """
     Class that takes care of converting unspecified data blob
     (Dict or List[Dict]) into datapoints (List[Dict]).
     If some properties that are required by Template class are missing
     ('x', 'y') it will attempt to fill in the blanks.
     """
 
-    @staticmethod
-    def update(datapoints: List[Dict], update_dict: Dict):
-        for data_point in datapoints:
-            data_point.update(update_dict)
-        return datapoints
-
```

Comment:
Lets keep that and its usage, this is one of the functionalities created for viewer. Also, the fact that it was used outside of `convert` was intentional.

Reply:
Alternative would be to consider updating `filename` and `revision` as steps that could be turned off, since we are using this mechanism.

Reply:
Updated the P.R. making `filename` and `revision` optional.

Also opened draft P.R. on studio https://github.com/iterative/studio/pull/3342 (locally I get tests to pass for studio)